content/docs/create-a-new-react-app.md

```diff
@@ -1,94 +1,94 @@
 ---
 id: create-a-new-react-app
-title: Create a New React App
+title: Crie um novo React App
 permalink: docs/create-a-new-react-app.html
 redirect_from:
   - "docs/add-react-to-a-new-app.html"
 prev: add-react-to-a-website.html
 next: cdn-links.html
 ---
 
-Use an integrated toolchain for the best user and developer experience.
+Use um toolchain integrado para uma melhor experiência de usuário e desenvolvedor.
```

Comment:
```suggestion
Use uma toolchain integrada para uma melhor experiência de usuário e desenvolvedor.
```
I would translate like this, since it means "uma cadeia de ferramenta".

Reply:
I just read the rest of the file and I would keep it consistent.

Reply:
Haha I had JUST pushed those changes when you commented. Thanks though!